portfolio/src/main/webapp/script.js

```diff
@@ -60,4 +223,20 @@ var unusedFunFacts =
     'Currently, I am attempting to grow potatoes outside.',
     'My favorite animal is a panda.'];
 
-var countTableEntries = 0;
+var commentTableText = `<table id="comment-table"> 
+            <tr> 
+                <th>Comments</th> 
+            </tr>`;
+
+var commentsAdded = 0;
+
+ /*
+  globalNumComments (for the client) stores the number of comments the user has selected to display for the particular section they are displaying/hiding/posting to.
+  The reason this variable has a default value of 10 is:
+  - 10 is a reasonable amount of comments to want to display on the page. 10 comments do not take up too much space on the page while still giving the user a 
+  good picture of what people are commenting.
+  - It is optional for the user to select the number of comments to display on the page, so we need a default value in case the user does not choose 
+  to select the number of comments to display.
+ */
+var globalNumComments = 10;
```

Comment:
I think a better name would be defaultNumComments. It is true its a global variable, but its function is that its a default value.